tests/models/test_dagrun.py

```diff
@@ -1638,8 +1625,8 @@ def double(value):
 
         @dag.task
         def consumer(value):
-            nonlocal result
-            result = list(value)
+            ...
+            # result = list(value)
```

Comment:
Intentional?

Reply:
Yeah, "this task is never run, but here's what it 'would' do"